Cargo.lock

```diff
@@ -4023,7 +4023,8 @@ checksum = "957e51f3646910546462e67d5f7599b9e4fb8acdd304b087a6494730f9eebf04"
 [[package]]
 name = "unique_port"
 version = "0.2.1"
-source = "git+https://github.com/ales-tsurko/iroha2-unique_port?rev=09e95bb3608626c995b8e7e51c64ba7486c88948#09e95bb3608626c995b8e7e51c64ba7486c88948"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "d3f903132efabcb4242e0bbf16aedb2c3867526e42126b6ed3cb1b0bc4577708"
```

Comment:
This is not my work, automatically done when I save files. I'm finding a way to avoid this

Reply:
It's ok. The crate was released on crates.io, so cargo changed the path. The path to my fork was an artifact.

Reply:
I don't think that this is a problem. 

First of all, the registry address should have been updated in the PR today. 

Secondly, the lockfile should be up to date. Just because it's done by Rust analyzer and not manually doesn't mean that it's wrong. 

> I'm finding a way to avoid this

While you could just ignore the `cargo.lock` file, I'd recommend that you actually keep committing the updated lockfile. It can save us a lot of troubleshooting.